RGO.Services.Tests/Services/EmployeeAddressServiceUnitTest.cs

```diff
@@ -59,6 +59,18 @@ public async Task CheckIfExistsPassTest()
         Assert.True(result);
     }
 
+    [Fact]
+    public async Task CheckIfExistPassFail()
```

Comment:
Just one thing does the name of this function make sense?

Reply:
No really ill change it to this (CheckIfExistsFail)